yt/frontends/__init__.py

```diff
@@ -0,0 +1,57 @@
+__all__ = [
+    "adaptahop",
+    "ahf",
+    "amrvac",
+    "art",
+    "arepo",
+    "artio",
+    "athena",
+    "athena_pp",
+    "boxlib",
+    "cf_radial",
+    "chimera",
+    "chombo",
+    "cholla",
+    "eagle",
+    "enzo_e",
+    "enzo",
+    "exodus_ii",
+    "fits",
+    "flash",
+    "gadget",
+    "gadget_fof",
+    "gamer",
+    "gdf",
+    "gizmo",
+    "halo_catalog",
+    "http_stream",
+    "moab",
+    "nc4_cm1",
+    "open_pmd",
+    "owls",
+    "owls_subfind",
+    "ramses",
+    "rockstar",
+    "sdf",
+    "stream",
+    "swift",
+    "tipsy",
+    "ytdata",
+]
+
+from functools import lru_cache
+
+
+@lru_cache(maxsize=None)
+def __getattr__(value):
+    import importlib
+
+    if value == "_all":
```

Comment:
maybe cause i'm sleep deprived, but took me a few minutes to realize this was a recursive call and i was very confused for a while lol. Mind adding the following comment (or something similar)?

```suggestion
    if value == "_all":
        # recursively import all frontends
```

Reply:
sure !